src/app/index.ts

```diff
@@ -0,0 +1,1003 @@
+
+import { get } from 'https';
+import Products from './product';
+
+let products: any = [];
+let loadingQuantity = 4;
+let catalogLength: any = null;
+let filteredProducts: any[] = [];
+const catalogList = document.querySelector(".catalog-list");
+const catalogListPopup = document.querySelector(".catalog-popup");
+const catalogLoadMore = document.querySelector(".catalog__more");
+
+
+class Cart {
+    public goods: any;
+    constructor() {
+        this.goods = [];
+    }
+
+
+    addProduct(id: number, name: string, quantity = 1) {
```

Comment:
it is good practice to specify the return type of a method or function. In this case, you can use `:void`